tests/test_tools_pdf2txt.py

```diff
@@ -47,11 +47,15 @@ def test_6(self):
     def test_7(self):
         run('../samples/contrib/','stamp-no')
     """
- 
+
     def test_8(self):
         run('../samples/contrib/','2b','-A -t xml')
 
     def test_9(self):
         run('../samples/nonfree/','175') # https://github.com/pdfminer/pdfminer.six/issues/65
+
+    def test_10(self):
+        run('../samples/scancode/','patchelf.pdf') # https://github.com/euske/pdfminer/issues/96
```

Comment:
Sounds like the test environment adds a `.pdf` extension for you so this should be `run('../samples/scancode/','patchelf')` instead